src/class-parsely.php

```diff
@@ -492,6 +487,31 @@ public function construct_parsely_metadata( array $parsely_options, WP_Post $pos
 		return apply_filters( 'wp_parsely_metadata', $parsely_page, $post, $parsely_options );
 	}
 
+	/**
+	 * Sets all metadata values related to post time.
+	 *
+	 * @param array   $metadata Array containing all metadata.
+	 * @param WP_Post $post Post object from which to extract time data.
+	 * @return void
+	 */
+	private function set_metadata_post_times( array &$metadata, WP_Post $post ): void {
+		// Initializations.
+		$date_format   = 'Y-m-d\TH:i:s\Z';
+		$post_time     = get_post_time( 'U', true, $post );
```

Comment:
As per https://developer.wordpress.org/reference/functions/get_post_time/ - this function can return `false`, so a guard needs adding here.